packages/tracing-processing/.env

```
@@ -0,0 +1,13 @@
+LOG_LEVEL=info
+
+DB_HOST=127.0.0.1
+DB_NAME=tracings
+DB_USERNAME=myusername
+DB_PASSWORD=mypwd
+DB_PORT=5432
+
+SCHEMA_NAME=myschemaname
+APPLICATION_NAME=pagopa-interop-tracing-operations
+
+SQS_ENDPOINT_CONSUMER="https://sqs.example.com/poll"
+SQS_ENDPOINT_PRODUCER="https://sqs.example.com/poll"
```

may we use more meaningful names?
are they
```suggestion
SQS_TRACING_UPLOD_ENDPOINT="https://sqs.example.com/poll"
SQS_PROCESSING_ERROR_ENDPOINT="https://sqs.example.com/poll"
```
?